Tests/Unit/LoRaTools/LnsRemoteCallTests.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
There's an identical test file in a different directory, we should remove it from there (similar to removing the `LnsRemoteCall` duplciated file).